all-clouds/all-clouds-explanation/cloud-init-metapackages.rst

```diff
@@ -0,0 +1,21 @@
+Cloud-init metapackages
+=======================
+
+Starting in Ubuntu Oracular (24.10), `cloud-init <https://launchpad.net/ubuntu/+source/cloud-init>`_
+added cloud-specific metapackages to better express cloud-specific dependencies and reduce image size.
```

Comment:
I think a little explanation will be useful here. Maybe change it to something like the following (please correct any word or statement that is inaccurate, I am just trying to give an idea of the content needed):

> added some metapackages to better express cloud-specific dependencies and to reduce image size.

>The original ``cloud-init`` is still available as a metapackage and a smaller version (called ``cloud-init-base``) has been created to include just the essential dependencies. A few specific metapackages have been created, while more cloud-specific ones are under development. The current set include:

Reply:
Done.